Sources/System/FileOperations.swift

```diff
@@ -168,7 +168,7 @@ extension FileDescriptor {
     retryOnInterrupt: Bool = true
   ) throws -> Result<Int, Errno> {
     valueOrErrno(retryOnInterrupt: retryOnInterrupt) {
-      system_read(self.rawValue, buffer.baseAddress, buffer.count)
+      Int(system_read(self.rawValue, buffer.baseAddress, numericCast(buffer.count)))
```

Comment:
Ideally, this casting would be done in the body of `read/write` inside the Windows syscall adapters. Can we avoid this at all?